### DISCUSSION TITLE
Cada que guardo una Kata se borra el codigo que hice

### DISCUSSION THREAD

**Discussion by CarIosLopez** [LaunchX-InnovaccionVirtual/CursoIntroPython#248] [General]
![image](https://user-images.githubusercontent.com/99137947/153960947-d9c0c324-372a-4d50-a8f4-cf8ab7c08d6a.png)

Tal cual dice el titulo, cada que guardo se borra todo lo que he creado, lo volví a crear todo desde un inicio y sigue borrándose

**Comment by HugoGuzmanPoxtan** [comment]
¿Como estas guardando tus archivos? O, ¿en definitiva no aparecen en tus documentos tus archivos creados?

**Reply by CarIosLopez** [reply]
Siempre las guardo al principio cuando la creo (antes de escribir código) pero cuando vuelvo a guardar un archivo después de escribir algo en la hoja me borra como el 80 - 90 % del código, **si aparecen los archivos pero el código se borra**

**Reply by carlos-gmx3** [reply]
Pero como la haces? yo presiono Ctrl + S

**Reply by CarIosLopez** [reply]
Ctrl + S y desde File - Save.

**Reply by carlos-gmx3** [reply]
pues no lo entiendo. tal vez algún archivo quedo mal al momento de la instalación, solo se me ocurre que los desinstales, reinicias y vuelves a instalar

**Reply by karen5454** [reply]
de preferencia ten activado el Auto Save
